src/server/game/World/World.cpp

```diff
@@ -2671,6 +2671,12 @@ BanReturn World::BanAccount(BanMode mode, std::string const& nameOrIP, uint32 du
     PreparedQueryResult resultAccounts = PreparedQueryResult(nullptr); //used for kicking
     PreparedStatement* stmt = nullptr;
 
+    // Prevent exploit with existing account ban
```

Comment:
what's the exploit ?

Reply:
https://github.com/TrinityCore/TrinityCore/issues/7466